scst/include/backport.h

```diff
@@ -1505,7 +1506,9 @@ static inline unsigned int scsi_prot_interval(struct scsi_cmnd *scmd)
  */
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 19, 0) || \
 	(defined(RHEL_RELEASE_CODE) && \
-	 RHEL_RELEASE_CODE -0 != RHEL_RELEASE_VERSION(8, 7)))
```

Comment:
Could you just change the condition from `!=` to `<` instead of duplicating this line for version 8.8.